### ISSUE TITLE
Blobs currently not supported/handled in Browser-Version

### ISSUE THREAD

**Issue by TheBlune** [naptha/tesseract.js#313]
I was trying to recognize text of a blob in the browser and got: 
`Uncaught (in promise) undefined`

that led me to this line in loadImage(): https://github.com/naptha/tesseract.js/blob/9a4f0b7eaa8129a1191ebf6c0082ac3b2fde31e1/src/browser/index.js#L81
Apparently the function does not check for Blobs. As there are only minified versions of Tesseract.js on CDN, I couldn't test any further...but the differing image-type-lists [here](https://github.com/naptha/tesseract.js/blob/master/docs/image-format.md) and [here](https://www.npmjs.com/package/tesseract.js#imagelike) and the function() suggest, that Blob support may already be possible. 

I am preparing a pull request to check for Blobs in LoadImage().
If Blobs are not supported yet, pls consider to at least implement a rejection handler for the rejection above, that throws an error like "Image Type currently not supported by Tesseract.js"
Also: Please consider providing a not-minified version of Tesseract on CDN or for download, so that we can quickly test changes to Tesseract.js (Or how would I go about in order to get an editable Copy of the Browser-Version of Tesseract.js?)

My original Code:
```
imageCapture.takePhoto()
        .then(blob => {
          loadLang({ langs: 'deu', langPath: worker.options.langPath })
           .then(() => {
             worker
               .recognize(blob)
               .progress(p => console.log(p))
               .then((result) => {
                 console.log(result.text);
                 worker.terminate();
               },(error) => {
                 console.log(error);
               })
           });
        })
```

Thanks!

**Comment by TheBlune** [comment]
That was easy! :)

But please consider reopening this issue, as you did not release the fix on CDN yet.

As I said: I am currently not able to test anything, as there are no not-minified versions of tesseract.js available. So currently I have to wait for your next npm-release, until I can use the fix for my development. Please don't let me wait approximately another 7 days...your library is so great that I can't wait to use it! :) 

Thanks.

**Comment by jeromewu** [comment]
Released v2.0.0-alpha.13 😄 
Enjoy the new feature and feel free to let me know if there is any other issues.
